Autofix unit test fails on Windows

Using version 1.5.1. (Well, this is really a development issue, no functional impact to consumers.)

Test "CLIEngine executeOnFiles() Fix Mode should return fixed text on multiple files when in fix mode" fails in my environment. In the `output` properties, the linefeeds are `\n`, but that code produces `\r\n` in my environment.

Output (edited somewhat for readability):

>   1) CLIEngine executeOnFiles() Fix Mode should return fixed text on multiple files when in fix mode:
> 
>       AssertionError: expected { Object (results, errorCount, ...) } to deeply equal { Object (results, errorCount, ...) }
>       + expected - actual
> 
>            {
>              "errorCount": 0
>              "filePath": "C:\\Users\\kpartington\\AppData\\Local\\Temp\\eslint\\fixtures\\fixmode\\quotes-semi-eqeqeq.js"
>              "messages": []
>       -      "output": "var msg = \"hi\";\r\nif (msg === \"hi\") {\r\n\r\n}\r\n"
>       +      "output": "var msg = \"hi\";\nif (msg === \"hi\") {\n\n}\n"
>              "warningCount": 0
>            }
>            // Snip
>              ]
>       -      "output": "var msg = \"hi\" + foo;\r\n"
>       +      "output": "var msg = \"hi\" + foo;\n"
>              "warningCount": 0
>            }
>          ]
>          "warningCount": 0
>       
>       at Function.assert.deepEqual (c:\Users\kpartington\Documents\GitHub\eslint\node_modules\chai\lib\chai\interface\assert.js:205:32)
>       at Context.<anonymous> (c:\Users\kpartington\Documents\GitHub\eslint\tests\lib\cli-engine.js:731:24)

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/27017077-autofix-unit-test-fails-on-windows?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F282608&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/tests/lib/cli-engine.js
+++ b/tests/lib/cli-engine.js
@@ -736,6 +736,17 @@ describe("CLIEngine", function() {
         describe("Fix Mode", function() {
 
             it("should return fixed text on multiple files when in fix mode", function() {
+                /**
+                 * Converts CRLF to LF in output.
+                 * This is a workaround for git's autocrlf option on Windows.
+                 * @param {object} result - A result object to convert.
+                 * @returns {void}
+                 */
+                function convertCRLF(result) {
+                    if (result && result.output) {
+                        result.output = result.output.replace(/\r\n/g, "\n");
+                    }
+                }
 
                 engine = new CLIEngine({
                     useEslintrc: false,
@@ -749,6 +760,7 @@ describe("CLIEngine", function() {
                 });
 
                 var report = engine.executeOnFiles([fixtureDir + "/fixmode"]);
+                report.results.forEach(convertCRLF);
                 assert.deepEqual(report, {
                     "results": [
                         {

